src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -111,6 +146,25 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
         this.addressBookFilePath = addressBookFilePath;
     }
 
+    /**
+     * Reset block and room values if they are invalid
+     */
+    public void validationCheck() {
+        if (minFloorSettings > maxFloorSettings) {
+            minFloorSettings = DEFAULT_MIN_FLOOR_SETTINGS;
+            maxFloorSettings = DEFAULT_MAX_FLOOR_SETTINGS;
+        }
```

Comment:
Need to handle the case when `minFloorSettings` < `maxFloorSettings`, but `maxFloorSettings` is invalid.

E.g. `minFloorSettings` = 99, `maxFloorSettings` = 100
After `validationCheck()`, `minFloorSettings` = 99, `maxFloorSettings` = 4

Reply:
its handled in the respective set methods, eg . `setMinFloorSettings`